guides/throttle.md

```diff
@@ -1,13 +1,17 @@
 ## API
 
-See `amoc_throttle`
+See `m:amoc_throttle`.
```

Comment:
please update the whole document and remove references to non-existing parameters (Rate, Interval, Interarrival, etc.). use corresponding atom keys in the configuration map instead.